src/eva/plotting/emcpy/diagnostics/vertical_line.py

```diff
@@ -4,7 +4,32 @@
 import os
 
 
+# --------------------------------------------------------------------------------------------------
+
+
 class VerticalLine():
+    """Creates a vertical line plot based on the provided configuration.
```

Comment:
Super minor thing -- in other files the """ is on a separate line.

Reply:
Great catch, thank you!